rmgpy/rmg/model.py

```diff
@@ -1343,31 +1343,13 @@ def getStoichiometryMatrix(self):
                 nu = rxn.getStoichiometricCoefficient(spec)
                 if nu != 0: stoichiometry[i,j] = nu
         return stoichiometry.tocsr()
-
-    def addSeedMechanismToCore(self, seedMechanism, react=False):
+    
+    def constructLibraryReactions(self,library):
         """
-        Add all species and reactions from `seedMechanism`, a 
-        :class:`KineticsPrimaryDatabase` object, to the model core. If `react`
-        is ``True``, then reactions will also be generated between the seed
-        species. For large seed mechanisms this can be prohibitively expensive,
-        so it is not done by default.
+        adds reactions from a given kinetics library in preparation to add it to core as seed mechanism
+        or to the edge as a reaction library
```

Comment:
From the comment, I don't understand:

1. what is the method adding reactions to if not to the core nor edge?
1. Is the attribute `library` a string or a library object?

It seems like only one line of this method requires the CoreEdgeReactionModel attributes. I think it might be better to have this method in library.py, and have it return a list of reactions. Then `addSeedMechanismToCore` can use `makeNewReactions`. This can also reduce the bloating, and subsequently difficulty in testing, of model.py that we have seen recently. 

When it is in library.py, it might be good to have a test method for it, unless you are hoping to scrap it soon and no longer rely on kinetics comments anymore.